### PR TITLE
[SDP-970][#102] Add Twilio SendGrid Email Client 

### PR DESCRIPTION
**Pull request by marwen-abid**
### What
* Adding Twilio SendGrid as an Email Client.

### Why
* Provide other alternatives to AWS. 
https://github.com/stellar/stellar-disbursement-platform-backend/issues/102

#### PR Structure

* [x] This PR has a reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR title and description are clear enough for anyone to review it.
* [x] This PR does not mix refactoring changes with feature changes (split into two PRs otherwise).

#### Thoroughness

* [x] This PR adds tests for the new functionality or fixes.
* [x] This PR contains the link to the Jira ticket it addresses.

#### Configs and Secrets

* [x] No new CONFIG variables are required -OR- the new required ones were added to the helmchart's [`values.yaml`] file.
* [x] No new CONFIG variables are required -OR- the new required ones were added to the deployments ([`pr-preview`], [`dev`], [`demo`], `prd`).
* [x] No new SECRETS variables are required -OR- the new required ones were mentioned in the helmchart's [`values.yaml`] file.
* [x] No new SECRETS variables are required -OR- the new required ones were added to the deployments ([`pr-preview secrets`], [`dev secrets`], [`demo secrets`], `prd secrets`).

#### Release

* [x] This is not a breaking change.
* [x] **This is ready for production.**. If your PR is not ready for production, please consider opening additional complementary PRs using this one as the base. Only merge this into `develop` or `main` after it's ready for production!

#### Deployment

* [ ] Does the deployment work after merging?

[`values.yaml`]: ../helmchart/sdp/values.yaml
[`pr-preview`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/stellar-disbursement-platform/backend-helm-values
[`dev`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/backend-helm-values
[`demo`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-backend-helm-values
[`pr-preview secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/externalsecrets-common-previews.yaml#L241-L346
[`dev secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/stellar-disbursement-platform-externalsecrets.yaml
[`demo secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-sdp-externalsecrets.yaml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,9 +118,10 @@ The Message Service sends messages to users and recipients for the following rea
 - Providing one-time passcodes (OTPs) to recipients
 - Sending emails to users during account creation and account recovery flows
 
-Note that the Message Service requires that both SMS and email services are configured. For emails, AWS SES is supported. For SMS messages to recipients, Twilio is supported. AWS SNS support is not integrated yet.
+Note that the Message Service requires that both SMS and email services are configured. For emails, AWS SES and Twilio Sendgrid are supported. For SMS messages to recipients, Twilio is supported. AWS SNS support is not integrated yet.
```
Comment (marcelosalloum) [inline README.md:121]: AWS SNS is supposed to work, right? I believe this phrase is outdated.

File: cmd/serve.go [hunk 2]
```diff
@@ -150,6 +150,7 @@ func (s *ServerService) SetupConsumers(ctx context.Context, o SetupConsumersOpti
 			MessageDispatcher:           o.ServeOpts.MessageDispatcher,
 			MaxInvitationResendAttempts: int64(o.ServeOpts.MaxInvitationResendAttempts),
 			Sep10SigningPrivateKey:      o.ServeOpts.Sep10SigningPrivateKey,
+			CrashTrackerClient:          o.ServeOpts.CrashTrackerClient.Clone(),
```
Comment (marcelosalloum) [inline cmd/serve.go:153]: 👀  Great catch!

**Review by marcelosalloum — APPROVED** [review]
LGTM! 👏

File: README.md (hunk 1 above)
Comment (marwen-abid) [reply README.md]: Addressed in [50bdd62](https://github.com/stellar/stellar-disbursement-platform-backend/pull/444/commits/50bdd621e9fb9806e27310a11d4ce411b09414be)

**Pull request merged**